cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -817,12 +817,12 @@ def _add_storage_security_group(self, storage_cfn_id, storage):
                 )
 
                 if sg_type == "Storage":
-                    ingress_rule.cfn_options.deletion_policy = ingress_rule.cfn_options.update_replace_policy = (
-                        storage_deletion_policy
-                    )
-                    egress_rule.cfn_options.deletion_policy = egress_rule.cfn_options.update_replace_policy = (
-                        storage_deletion_policy
-                    )
+                    ingress_rule.cfn_options.deletion_policy = (
+                        ingress_rule.cfn_options.update_replace_policy
```

Comment:
Did you fix this manually?

Reply:
Nope these are automatically changing due to Tox